adb_graphics/datahandler/gribdata.py

```diff
@@ -613,10 +613,11 @@ def grid_info(self):
             attrs = ['Lov']
             grid_info['projection'] = 'stere'
             grid_info['lat_0'] = 90
-            grid_info['lat_ts'] = 90
         elif self.grid_suffix == 'GLL0':
             attrs = []
             grid_info['projection'] = 'cyl'
+            grid_info['lon_0'] = 0
+            grid_info['lat_0'] = 90
```

Comment:
Are these set in the file's metadata, or no?

Reply:
I tested with these two lines commented out, relying on the default values.  Everything ran perfectly, and the resulting plots looked the same.  I removed the lines.